.circleci/config.yml

```diff
@@ -36,13 +38,13 @@ aliases:
 
   - &run_tests
     name: run_tests
+    no_output_timeout: 60m
```

Comment:
just curious, what's the default?

Reply:
default time out is 10m.
git clone of uvcdat-testdata sometimes take looong..